README.md

```diff
@@ -40,14 +40,14 @@ If you choose `WIFI_ESP8266`, you'll also need to add the WiFi SSID and password
             "help": "Pin used as RX (connects to ESP8266 TX)",
             "value": "PTD2"
         },
-        "esp8266-ssid": {
+        "esp8266-debug": {
```

Comment:
It enables tracing of the AT commands of the module, very useful for debugging. It has to be configured in the constructor, so this is a quick way of communicating from app -> the esp8266 driver.